src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -38,7 +72,16 @@ export const CollectorProfileSaves2RouteFragmentContainer = createFragmentContai
   {
     me: graphql`
       fragment CollectorProfileSaves2Route_me on Me {
-        collectionsConnection(first: 20) {
+        defaultSaves: collection(id: "saved-artwork") {
+          internalID
+          ...SavesItem_item
+        }
+
+        otherSaves: collectionsConnection(
+          first: 20
```

Comment:
**suggestion:** Maybe we can update this to 30 — since we've [decided on 30 as our hard-coded limit](https://www.notion.so/artsy/Artwork-lists-decisions-and-glossary-2715a5edf2fa4fea8797d2b51bd6214f) for v1 of this feature

Reply:
done